paf-mvp-frontend/src/containers/welcome-widget/WelcomeWidget.tsx

```diff
@@ -32,7 +32,7 @@ export const WelcomeWidget = ({ emitConsent }: IWelcomeWidgetProps) => {
 
   const originalIdentifier = originalData?.identifiers?.[0];
   const originalConsent = originalData?.preferences?.data?.use_browsing_for_personalization;
-  const pafNodeHost = currentScript.getData()?.proxy;
+  const pafClientNodeHost = currentScript.getData()?.proxy;
```

Comment:
Not sure that the renaming is worth it.

Reply:
The point is:
1. "node" in itself doesn't mean much, while "client node" puts it perspective
2. "client node host" is what is used elsewhere. Ideally, I would have updated `proxyHostName`  as well, but since it is the public facing API I want to preserve it for the moment